### ISSUE TITLE
Spike: Make logging ILogger<> compatible

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#14]
### Expected result

Can we make the [Xperience event logging](https://docs.kentico.com/13/securing-websites/developing-secure-websites/handling-error-messages-securely/designing-secure-error-messages#Designingsecureerrormessages-Storingthedebugandtraceinformationintheeventlog) accessible as an [ILogger implementation](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-3.1)?

### Additional context

As it stands now, the Xperience logging is accessible as a service that's resolved at a later stage in the app startup. Events that occur at app startup won't be logged in Xperience (e.g. at each [recycle](https://amanagrawal.blog/2019/09/07/iis-app-pool-recycles-will-wreck-your-long-running-background-jobs/)). Having an ILogger<> implementation would unify logging into one.

### Resources

* https://asp.net-hacker.rocks/2017/05/05/add-custom-logging-in-aspnetcore.html

### Issue expands upon

This issue expands upon another issue(s): #17

### Blockers

This issue is blocked by the following issue(s): #`issue number`
